app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -353,7 +353,8 @@ fun EditBillAmounts() {
                     modifier = Modifier
                         .width(100.dp)
                         .height(55.dp),
-                    textStyle = TextStyle.Default.copy(fontSize = 18.sp)
+                    textStyle = TextStyle.Default.copy(fontSize = 18.sp, fontWeight = FontWeight.Bold)
+
```

Comment:
As we created the box the content in the box will be bolded

Reply:
This appears to be a lot of net new.  I'm not entirely clear on what's going on here.